roles/frontend/defaults/main.yml

```diff
@@ -2,6 +2,8 @@ frontend_dehydrated_domains:
  - "{{ frontend_nginx_ssl_name }}"
 frontend_nginx_api_servers:
  - apps.flathub.org:8080
+frontend_nginx_prerender_servers:
+ - 45.55.104.129:3000
```

Comment:
Is there any real benefit to using your beta server (and hence introducing a production dependency on it) versus using the public service on prerender.io? Or could we use both?

Reply:
Prerender.io free tier is for 250 pages or less. We already have more
https://prerender.io/pricing